z2/Cargo.toml

```diff
@@ -43,7 +43,7 @@ itertools = "0.13.0"
 jsonrpsee = {version = "0.22.4", features = ["client"]}
 k256 = "0.13.3"
 lazy_static = "1.5.0"
-libp2p = {version = "0.54.0", features = ["identify"]}
+libp2p = {version = "0.54.0", features = ["identify", "autonat"]}
```

Comment:
Isn't this activating the feature in the wrong `Cargo.toml`? Should this move to `zilliqa/Cargo.toml`?

I guess this still works, as long as you `cargo build` at the root of the workspace, since the features get unified anyway.

Reply:
Yep, my bad, I'll switch that over